test/emqx_client_SUITE.erl

```diff
@@ -49,6 +58,62 @@ init_per_suite(Config) ->
 end_per_suite(_Config) ->
     emqx_ct_broker_helpers:run_teardown_steps().
 
+request_response_test(_Config) ->
```

Comment:
there are 3 tests for QoS-0/1/2 each, likely copy-pasting from a quick glance.
please consider using a function with QoS parameterized.